app/templates/view_buyers.html

```diff
@@ -53,7 +53,13 @@ <h1 class="govuk-heading-xl">Find a buyer</h1>
 
       {% call summary.row() %}
 
-        {{ summary.field_name(item.name) }}
+        {% call summary.field() %}
+          {% if current_user.has_any_role('admin') %}
+            <a class="govuk-link" href="{{ url_for('.change_user_name', user_id=item.id) }}">{{ item.name }}</a>
```

Comment:
an edit button would probably be more intuitive but agree a link makes the most sense here :+1:

Reply:
Yes. I wanted to make the smallest possible change - there are a lot of things that one could edit here.